openshift/templates/build.bc.yaml

```diff
@@ -32,7 +32,7 @@ parameters:
   - name: DOCKER_IMAGE
     description: Location of docker image
     required: true
-    value: image-registry.openshift-image-registry.svc:5000/e1e498-tools/wps-api-base:ubuntu.22.04-latest
+    value: image-registry.openshift-image-registry.svc:5000/e1e498-tools/wps-api-base:02-10-2024
```

Comment:
Note: not going to merge as is, once approved will retag `wps-api-base:02-10-2024` as `wps-api-base:ubuntu.24.04-latest`